### PR TITLE
create Avatar component

### PR DESCRIPTION
**Pull request by 888yalo888**
Created an Avatar component, review it please

### CODE REVIEW

File: src/types/avatarTypes.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export enum Health {
+  full = 'full',
+  damaged = 'damaged',
+  nearlydead = 'nearlydead',
+  dead = 'dead',
+}
+
+export enum selection {
```
Comment (DenielNCore) [inline src/types/avatarTypes.ts]: Wrong case

File: src/types/avatarTypes.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export enum Health {
```
Comment (DenielNCore) [inline src/types/avatarTypes.ts]: Can be changed to avatar health

File: src/components/Skills.vue [hunk 3]
```diff
@@ -10,7 +12,18 @@
 </script>
 
 <template>
-  <div>skills</div>
+  <Avatar :health="Health.full" :selection="selection.none" title="5" />
```
Comment (DenielNCore) [inline src/components/Skills.vue]: To be removed this file changes

File: src/components/Avatar.vue [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  const props = defineProps<{
+    health: Health;
+    selection: selection;
+    title: string;
+  }>();
+</script>
+
+<template>
+  <div class="avatar-container" :class="{ unselected: props.selection === 'none' }">
+    <img :src="Avatar" v-if="props.selection === selection.current" class="current" />
```
Comment (DenielNCore) [inline src/components/Avatar.vue]: Replace with computed class

File: src/components/Avatar.vue [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    <img :src="Avatar" v-if="props.selection === selection.selected" class="selected" />
+    <img v-if="props.selection === selection.none" :src="Avatar" />
```
Comment (DenielNCore) [inline src/components/Avatar.vue]: You should simplify this ifs

File: src/types/avatarTypes.ts (hunk 2 above)
Comment (888yalo888) [reply src/types/avatarTypes.ts]: changed

**Review by DenielNCore — APPROVED** [review]

**Review by DenielNCore — APPROVED** [review]

**Pull request merged**